src/core/net/dns_types.cpp

```diff
@@ -564,6 +564,11 @@ Error Name::LabelIterator::ReadLabel(char *aLabelBuffer, uint8_t &aLabelLength,
     return error;
 }
 
+bool Name::LabelIterator::LowercaseMatch(uint8_t aFirst, uint8_t aSecond)
```

Comment:
```suggestion
bool Name::LabelIterator::CaseInsensitiveMatch(uint8_t aFirst, uint8_t aSecond)
```

I think comparing case insensitively is the actual purpose.

Reply:
Good suggestion. Will change it in next push.

Reply:
Updated in new push.